Changed design of website

Zibethin opened the pull request:
Changed images, font, background, colours and further formatting, content and layout to be more user friendly

Zibethin committed:
changed background image for intro page into image with Red cross worker, added the image to directory, added a higher resolution image of the logo for relief-web, changed css to align card for external and internal resources, changed background colour between sections and changed main font.

pknight03 committed:
Change file name (header)

Zibethin committed:
Merge pull request #1 from pknight03/patch-1

Change file name (header) for front-banner

Zibethin committed:
Testing Git synchronization

Zibethin committed:
fixed image sizing compatibility with chrome, fixed box sizing to accommodate large font sizes

Zibethin committed:
edited images, changed card order, changed background and title of 1st section

Zibethin committed:
changed document section to be a list centered and aligned

Zibethin committed:
updated the pdfs to be EN language, changed colours and font size for intro page What is SIMS?

Zibethin committed:
fixing compatibility issue with logo sizing

Zibethin committed:
IE patch

pknight03 committed:
text change

pknight03 committed:
text change (2)

Zibethin committed:
Merge pull request #3 from pknight03/patch-3

text content change (2)

Zibethin committed:
Merge pull request #2 from pknight03/patch-2

text content change

danbjoseph commented:
thanks for this. on a whirlwind work trip right now but hope to have a chance to look over this soon.

Zibethin committed:
changed activation image

Zibethin committed:
Added Sims activations dashboard to page

added dc/d3/crossfilter, added data files, added dashboard.js, amended
html and css

Zibethin committed:
Changed all links to https

to not mix http and https

Zibethin committed:
Updated dashboard (table formatting, transitions)

Added loading button during page load,
fixed formating of the table (improved responsive sizing, fixed row
height),
fixed reset button moving around,
Changed formatting of reset button,
Automated max scale of palette

Zibethin committed:
fixed data location for dashboard

Zibethin committed:
fixed another location issue

Zibethin committed:
Added data download, fixed responsiveness

Added a download button for data, added fileSave.js to allow pop-up when
saving.
Added a box around map with scrolls, made map and table disappear on
small screen, text will appear instead.
Moved reset-button to center of map.
Removed loading button.

Zibethin committed:
Added pknight's pull request

Zibethin committed:
Fixed path

Zibethin committed:
fixed mobile card alignment

Zibethin committed:
Changed text from pknight

Zibethin committed:
Changed background colour to be white by default

dalekunce commented:
@Zibethin thanks for doing some of this work. I'm going to start reviewing it today. I'll try and provide specific comments when possible. 

One thing that I'm noticing right off the top is an introduction of whitespace and inconsistent tabs/spaces and single vs double quote use. When these are changed from the base code its very hard to tell what is actually changing.

dalekunce commented on css/sims.css:
```diff
@@ -1,3 +1,27 @@
+   text-shadow: -0.1em 0 white, 0 0.1em white, 0.1em 0 white, 0 -0.1em white;
+}
+
+.bg-white {
```
why not just put this on the body class?

Zibethin replied:
The text shadow? It's only for the title. If you mean the bg-white, I've changed that in a recent commit https://github.com/Zibethin/sims-website/commit/9ecdea826ad8b8c846c2f990025242c5ffe4a246

dalekunce commented on css/sims.css:
```diff
@@ -27,24 +51,46 @@
   margin-top: 20px;
-  box-sizing: border-box;
-  border-radius: 2px;
+  border-radius: 5px;
```
I don't like the new large rounded corners. 2px is a better way to go.

dalekunce commented on css/sims.css:
```diff
@@ -27,24 +51,46 @@
   background-clip: padding-box;
-  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+  overflow:hidden;
+  height:300px;
```
I liked the way the cards "fit" together in the current page. It requires less screen real estate and doesn't introduce so much unused whitespace on the page.

dalekunce commented on index.html:
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```
be careful to avoid introducing changes like this. Nothing changed!! Why is this getting shown.

Zibethin replied:
Thanks! Will be careful next time!

dalekunce commented on index.html:
```diff
@@ -14,6 +14,7 @@
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet"> 
```
Too many fonts on the page. OpenSans only please.

dalekunce commented on index.html:
```diff
@@ -64,18 +65,18 @@
 
     <!-- About -->
-    <section id="about" class="about">
+    <section id="about" class="about bg-white">
```
I don't like the zebra approach to the site now, white/black is to big of a contrast. If you want to try this try white and a soft grey or other soft color.

dalekunce reviewed (commented):
this kind of stuff should *not* show up in a commit history.

dalekunce reviewed (changes requested):
this makes the page much longer without adding to much value. Try putting this into a 2 or 3 up column

Zibethin commented:
Ah sorry before you review it, this code is not up to date could I pull through the new code for review?

dalekunce reviewed (commented):
+1

dalekunce commented:
@Zibethin feel free to keep pushing code here. I'm making some must changes before we can merge anyway.

dalekunce commented on data/SIMS Activation Log.csv:
```diff
@@ -0,0 +1,34 @@
+Year,Country,Code,Response,Support,Significance (1-5; 5 as most significant activations),Start Date,End Date
```
This should *not* be a static file on the website it will be out of date very quickly. Make this an AJAX call to a Google Sheet. Who is tasked with maintaining this? How are the severities decided?

Zibethin replied:
Yes I originally wanted to do that but wasn't sure how to do it. Happy to learn how to do it from you! The person maintaining the sheet and the severities is @pknight03, he can provide you with the original document :)

pknight03 replied:
Hi,

The spreadsheet is located in the SIMS Google Drive folder name: 'SIMS Activation Log'. It was originally started by Kat and has now passed on to me to maintain. I can ping you if you dont have the direct link. 

I decided not to do an AJAX call to the Google Sheets just yet as it contains names of people deployed etc. I agree this is the way to go instead of manually pulling through an edited csv each time. @simonbjohnson mentioned something about an intermediary step being thought about for this very reason. 

The severities were originally devised by our team sometime last year to explain SIMS commitment to other teams. They could be seen to be confusing to the general viewer and maybe should be  removed as do not add direct value. The levels were decided as (for the entire response):
5 - A full time person coordinating, deployed and remote for all/majority of the response
1 - Less than 1 month, equates to 1 day a week for 1 month, remote only.

dalekunce replied:
@pknight03 can you limit the scope to a google sheet that *only* has the data we want.

Zibethin committed:
Merge remote-tracking branch 'refs/remotes/origin/gh-pages' into AmericanRedCross/gh-pages

dalekunce commented on index.html:
```diff
@@ -400,58 +460,62 @@ <h4 class="no-margin"><strong>The World Bank</strong>&nbsp; <small>has a wealth
+
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="js/d3.js"></script>
```
lets move all vendor js and css into js/vendor/*.js

We might also want to start using bower to handle dependencies for us.

Zibethin committed:
- reduced whitespace in cards (changed responsiveness to work for small devices too)
- changed Oxygen font to Open-Sans
- changed dark background to be less dark

Zibethin committed:
- Moved all javascript files to a vendor folder

Zibethin committed:
- minified dc.js, d3.js, crossfilter.js, fileSaver.js into an all.min.js file (the package.json and gulpfile.js are used to minify the other files and are only included for reproduceability, they are not needed for the website)

dalekunce reviewed (commented):
👍 for using gulp this all looks good.

Zibethin committed:
changed the static csv to ajax call to google sheet

Zibethin committed:
Merge remote-tracking branch 'refs/remotes/origin/gh-pages' into AmericanRedCross/gh-pages

Zibethin committed:
Merge remote-tracking branch 'refs/remotes/origin/gh-pages' into gh-pages

Zibethin commented:
@dalekunce Hi Dale, I think the requests have addressed, would it be possible to get your feedback about them? Thanks!

Zibethin committed:
Resized images to be smaller
